docs/index.html

```diff
@@ -203,10 +203,17 @@ <h5>If you have any legal concerns regarding copyrights or want to report an abu
 
       <div class="search">
         <input type="text" placeholder="Search for icons" ng-model="search">
+        <input type="text" class="result" value="{{ filtered.length }}" title="Number of icons in the current search" disabled>
       </div>
 
+      <ul class="computed-data">
+        <li title="The number of icons. For example: devicon">Total icons = {{ icons.length }}</li>
+        <li title="The number of icon versions. For example: devicon-plain">Total icon version = {{ totalFonts }}</li>
+        <li title="The number of icon versions per icon. Calculated by dividing icon version by icons">Average icon versions per icon = {{ avgFonts }}</li>
```

Comment:
Just a style suggestion: replace equal signs to semicolons. 🙂
```suggestion
        <li title="The number of icons. For example: devicon">Total icons: {{ icons.length }}</li>
        <li title="The number of icon versions. For example: devicon-plain">Total icon version: {{ totalFonts }}</li>
        <li title="The number of icon versions per icon. Calculated by dividing icon version by icons">Average icon versions per icon: {{ avgFonts }}</li>
```

Reply:
Fixed: c9ace88